.github/ISSUE_TEMPLATE/release_issue_template.md

```diff
@@ -4,7 +4,7 @@
 Note to onduty:
 
 Use this issue to track a release from the initial canary release build through
-production. The community uses this issue to keep track of what is going on
+promotion to Stable. The community uses this issue to keep track of what is going on
```

Comment:
Why do we have this template twice? 🙃

Reply:
I don't know but I've been making sure to keep them in sync :man_shrugging:

Reply:
Good catch by @danielrozenberg. The file `.github/ISSUE_TEMPLATE/release_issue_template.md` is old, and can be deleted.

Reply:
To be 100% sure, here are screenshots that confirms that `.github/ISSUE_TEMPLATE/release-tracking-issue.md` is the file used as a template, while `.github/ISSUE_TEMPLATE/release_issue_template.md` is not.

![image](https://user-images.githubusercontent.com/26553114/72190088-a272d100-33cc-11ea-8473-de60af637778.png)
![image](https://user-images.githubusercontent.com/26553114/72190137-c9310780-33cc-11ea-890f-09b6539b29c3.png)

In addition, the [file history](https://github.com/ampproject/amphtml/commits/master/.github/ISSUE_TEMPLATE/release_issue_template.md) for `.github/ISSUE_TEMPLATE/release_issue_template.md` confirms that it hasn't been touched in almost a year.

Reply:
Great find! Removed.